contracts/source/recoverable-wallet.sol

```diff
@@ -261,14 +222,12 @@ contract RecoverableWallet is Ownable, Erc777TokensRecipient {
 
 	/// @notice finishes the recovery process after the necessary delay has elapsed.  callable by anyone in case the keys controlling the active recovery address have been lost, since once this is called a new recovery (with a potentially lower recovery priority) can begin.
 	function finishRecovery() external onlyDuringRecovery {
-		require(block.timestamp > activeRecoveryEndTime, "You must wait until the recovery delay is over before finishing the recovery.");
+		require(block.timestamp >= activeRecoveryEndTime, "You must wait until the recovery delay is over before finishing the recovery.");
```

Comment:
Specific thought here about >= ? To allow a single-tx to switch as pending owner replacement?

Reply:
Yeah, single transaction or single block transfer.  It is especially annoying in testing where `now` only moves forward when transactions are submitted.